<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWineYardDialog</class>
 <widget class="QDialog" name="NewWineYardDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>542</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>505</width>
    <height>542</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Wineyard</string>
  </property>
  <property name="windowIcon">
   <iconset theme="wine">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="KTitleWidget" name="wineyardInfoTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Basic Information</string>
   </property>
   <property name="icon">
    <iconset theme="wine">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoHideTimeout">
    <number>0</number>
   </property>
  </widget>
  <widget class="KLineEdit" name="wineyardNameField">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="toolTip">
    <string>Choose a name for your WineYard.</string>
   </property>
   <property name="statusTip">
    <string>Choose a name for your WineYard.</string>
   </property>
   <property name="whatsThis">
    <string>Choose a name for your WineYard.</string>
   </property>
   <property name="placeholderText">
    <string>Name of the WineYard</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="showClearButton" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="KComboBox" name="wineyardTypeField">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Type of WineYard (environment)</string>
   </property>
   <property name="autoCompletion">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Gaming</string>
    </property>
    <property name="icon">
     <iconset theme="input-gaming">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custom</string>
    </property>
    <property name="icon">
     <iconset theme="settings">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Application</string>
    </property>
    <property name="icon">
     <iconset theme="application-x-ms-dos-executable">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="wineyardVersioningCB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Enable Versioning</string>
   </property>
   <property name="icon">
    <iconset theme="time">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+V</string>
   </property>
  </widget>
  <widget class="KTitleWidget" name="wineyardOptionsTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Extra Options</string>
   </property>
   <property name="icon">
    <iconset theme="wine">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoHideTimeout">
    <number>0</number>
   </property>
  </widget>
  <widget class="KHistoryComboBox" name="runnerVersionField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="currentText">
    <string>Default</string>
   </property>
   <property name="placeholderText">
    <string>Runner Version</string>
   </property>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="runnerVersionLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Runner Version</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="wineyardTypeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>WineYard Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Create WineYard</string>
   </property>
   <property name="icon">
    <iconset theme="wine">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="discardButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Discard WineYard</string>
   </property>
   <property name="icon">
    <iconset theme="wine">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="KCapacityBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress</string>
   </property>
   <property name="barHeight">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KCapacityBar</class>
   <extends>QWidget</extends>
   <header>kcapacitybar.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
